genrules.cpp

```diff
@@ -320,8 +320,6 @@ string Game::FactionTypeDescription(Faction &fac) {
 		if (key == F_TRADE) buffer << "perform trade in " << nt << " " << plural(nt, "region", "regions");
 		if (key == F_MAGIC) {
 			buffer << "have " << nm << " " << plural(nm, "mage", "mages");
-			if (Globals->APPRENTICES_EXIST)
-				buffer << " as well as " << na << " " << Globals->APPRENTICE_NAME << (na > 1 ? "s" : "");
```

Comment:
Believe it or not, this was actually wrong here.  There is a ruleset (Havilah) which allows 1 apprentice at Magic 0.
So in order for that rule set to show correctly, the apprentices have to be handled irrespective of if a faction is magic which is done below.
Doing it here as well resulted in all rulesets double reporting apprentices.